<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="147dda8202cd.ant.amazon.com" failures="1" tests="6" name="API TestCases" time="36.901" errors="0" timestamp="13 Feb 2022 14:32:25 GMT">
  <testcase classname="api.com.tests.apiTestScript" name="validateResponseBodyPojoPost" time="30.471">
    <skipped/>
  </testcase> <!-- validateResponseBodyPojoPost -->
  <testcase classname="api.com.tests.apiTestScript" name="validateResponseBodyPojoPost" time="1.537">
    <skipped/>
  </testcase> <!-- validateResponseBodyPojoPost -->
  <testcase classname="api.com.tests.apiTestScript" name="validateResponseBodyPojoPost" time="1.391">
    <skipped/>
  </testcase> <!-- validateResponseBodyPojoPost -->
  <testcase classname="api.com.tests.apiTestScript" name="validateResponseBodyPojoPost" time="1.105">
    <skipped/>
  </testcase> <!-- validateResponseBodyPojoPost -->
  <testcase classname="api.com.tests.apiTestScript" name="validateResponseBodyPojoPost" time="1.019">
    <skipped/>
  </testcase> <!-- validateResponseBodyPojoPost -->
  <testcase classname="api.com.tests.apiTestScript" name="validateResponseBodyPojoPost" time="0.873">
    <failure type="java.lang.AssertionError" message="expected [200] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [201]at api.com.tests.apiTestScript.validateResponseBodyPojoPost(apiTestScript.java:193)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateResponseBodyPojoPost -->
</testsuite> <!-- API TestCases -->
